.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,121 @@
+name: Benchmark CI
+
+on:
+  pull_request:
```

Comment:
I don't think we want to run this job on PRs - it takes too long.

Reply:
Same for branches. I think the best would be a scheduled job, say once every 3 days.

Reply:
My idea here was to run this on specific PRs with a specific label, like `run-benchmarks`. It would be quite useful for significant PRs, especially around releases. Please let me know your thoughts, otherwise I would just remove this out.

Reply:
I like that idea. That's even better than regularly scheduled runs, very user-friendly.

Can you make the label `ci/run-benchmarks`? Then we can later extend the pattern with `ci/other-job-to-run`. PyTorch has this too, and it's quite nice.

Reply:
Sure, I will make this change.